### PR TITLE
Replace RCT2_GLOBAL magic numbers with address identifiers and string ids

### PR DESCRIPTION
**Pull request by Overv**
I used a script to replace all address constants and string ids in RCT2_GLOBAL invocations with their corresponding enum values - if available.

You can find the script here:

https://gist.github.com/Overv/064c95408203a49d95ef

### CODE REVIEW

**Comment by janisozaur** [comment]
oh cool, how about we use this tool in travis?

File: src/scenario.c [hunk 1]
```diff
@@ -817,7 +817,7 @@ int scenario_get_num_packed_objects_to_write()
 	rct_object_entry_extended *entry = (rct_object_entry_extended*)0x00F3F03C;
 
 	for (i = 0; i < 721; i++, entry++) {
-		if (RCT2_ADDRESS(0x009ACFA4, uint32)[i] == 0xFFFFFFFF || (entry->flags & 0xF0))
+		if (RCT2_ADDRESS(RCT2_ADDRESS_RIDE_ENTRIES, uint32)[i] == 0xFFFFFFFF || (entry->flags & 0xF0))
```
Comment (duncanspumpkin) [inline src/scenario.c]: 0x9ACFA4 is old format we should avoid using this. `GET_RIDE_ENTRY(i)` is preferred.

**Comment by Overv** [comment]
I have extended the script to handle those.

File: src/ride/ride.h [hunk 2]
```diff
@@ -858,7 +858,7 @@ extern const uint8 gRideClassifications[255];
 #define _currentTrackBeginX							RCT2_GLOBAL(0x00F440A8, uint16)
 #define _currentTrackBeginY							RCT2_GLOBAL(0x00F440AA, uint16)
 #define _currentTrackBeginZ							RCT2_GLOBAL(0x00F440AC, uint16)
-#define _currentTrackPieceDirection					RCT2_GLOBAL(0x00F440AE, uint8)
+#define _currentTrackPieceDirection					RCT2_GLOBAL(RCT2_ADDRESS_TRACK_PREVIEW_ROTATION, uint8)
```
Comment (IntelOrca) [inline src/ride/ride.h]: I'd revert this one, its a different way of doing RCT2 GLOBALs.

**Comment by IntelOrca** [comment]
That's fine, you need to rebase and then I can merge.

**Comment by Overv** [comment]
Done.

**Pull request merged**